### PR TITLE
Fix missing properties in Onto and Vocabs

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

Fixes missing properties

### CODE REVIEW

**Comment by ioggstream** [comment]
@giorgialodi quando puoi dare un'occhiata, in modo da testare l'harvesting.

File: VocabolariControllati/classifications-for-her/erc-panel-h2020-fp/erc-panel-h2020-fp.ttl [hunk 1]
```diff
@@ -202,6 +202,7 @@ ndc:keyConcept
       dcat:contactPoint <https://w3id.org/italia/data/contact-point/voc_AgID> ;
       dct:issued "2020-06-06"^^xsd:date ;
       dct:modified "2020-06-11"^^xsd:date ;
+      owl:versionInfo "0.2" ;
```
Comment (ioggstream) [inline VocabolariControllati/classifications-for-her/erc-panel-h2020-fp/erc-panel-h2020-fp.ttl:205]:
```suggestion
      owl:versionInfo "0.1" ;
```

File: VocabolariControllati/classifications-for-transparency/transparency-obligation/transparency-obligation.ttl [hunk 2]
```diff
@@ -53,7 +54,7 @@
         skos:hasTopConcept  :160101 , :020501 , :040202 , :190101 , :150201 , :150901 , :210201 , :010202 , :210105 , :150701 , :040101 , :150101 , :040901 , :140101 , :100102 , :010203 , :040701 , :160301 , :080101 , :050101 , :060301 , :030201 , :110101 , :100206 , :090101 , :080201 , :110201 , :120201 , :150301 , :020302 , :020102 , :090201 , :020106 , :130101 , :010201 , :041101 , :150401 , :100101 , :210104 , :070401 , :120101 , :120102 , :140201 , :100201 , :130201 , :100204 , :100205 , :070201 , :210303 , :210203 , :210103 , :160201 , :040301 , :060201 , :140301 , :060101 , :010301 , :150802 , :180101 , :010101 , :200101 , :020107 , :020301 , :020201 , :040801 , :170101 , :020101 , :150801 , :060401 , :150601 , :041001 , :040203 , :150501 , :210202 , :210302 , :010205 , :210102 , :020401 , :210106 , :040102 , :040501 , :100202 , :100203 , :030101 , :041201 , :210301 , :010204 , :040601 , :070301 , :070101 , :060501 ;
 
 
-
+        ndc:keyConcept "transparency-obligation" ;
```
[the remark this answers is not in the document]
Comment (giorgialodi) [reply VocabolariControllati/classifications-for-transparency/transparency-obligation/transparency-obligation.ttl:57]: Ok for this change

**Review by giorgialodi — APPROVED** [review]
Roberto, that's ok all. Also the first element is not an issue to include 0.2. Let's proceed.

**Pull request merged**